tests/validation/tests/rke/test_install_config.py

```diff
@@ -6,7 +6,7 @@ def test_install_config_1(test_name, cloud_provider, rke_client, kubectl):
     """
     Node Address specified as just IP and using only this in the node spec
```

Comment:
Could you add a comment that a specific K8s version is being tested in this test case ? We are replacing cluster_install_config_1.yml.j2 with cluster_install_config_11.yml.j2 . Do we still want to keep cluster_install_config_1.yml.j2 ?

Reply:
Done.
Yes `cluster_install_config_1.yml.j2` is still available as its being used by another test.